addons/website_sale/views/templates.xml

```diff
@@ -1834,9 +1836,13 @@
                     <t t-if="payment_tx_id.state == 'pending'">
                         <t t-out="payment_tx_id.acquirer_id.sudo().pending_msg"/>
                     </t>
-                    <t t-if="payment_tx_id.state == 'done'">
+                    <t t-if="payment_tx_id.state == 'done' and order.amount_total == payment_tx_id.amount">
                         <span t-if='payment_tx_id.acquirer_id.sudo().done_msg' t-out="payment_tx_id.acquirer_id.sudo().done_msg"/>
                     </t>
+                    <t t-if="payment_tx_id.state == 'done' and order.amount_total != payment_tx_id.amount">
+                        <span t-if='payment_tx_id.acquirer_id.sudo().done_msg' t-out="payment_tx_id.acquirer_id.sudo().done_msg"/>                     </t>
```

Comment:
Is it normal to show the done_msg here?

Reply:
well the transaction was done and successful so in a sens yes, but it's debatable
@dalaOdoo?

Reply:
true ;)

Reply:
it's important to mentioned that the payment was made yes. But it didn't confirm the sales because amount paid != total (due to a change in the cart during the payment process)